common/djangoapps/util/milestones_helpers.py

```diff
@@ -313,6 +314,28 @@ def remove_content_references(content_id):
     return milestones_api.remove_content_references(content_id)
 
 
+def any_unfulfilled_milestones(course_id, user_id):
+    """ Returns a boolean if user has any unfulfilled milestones """
+    if not settings.FEATURES.get('MILESTONES_APP', False):
+        return None
```

Comment:
You are returning None here but the code is expecting a bool -- I suppose it's a matter of semantics, but would returning False make more sense?